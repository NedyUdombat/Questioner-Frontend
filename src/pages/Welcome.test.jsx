import React from 'react';
import { shallow } from 'enzyme';

import Welcome from './Welcome';

describe('App', () => {
  it('should render correctly', () => {
    const component = shallow(<Welcome />);
    expect(component).toMatchSnapshot();
  });
});
