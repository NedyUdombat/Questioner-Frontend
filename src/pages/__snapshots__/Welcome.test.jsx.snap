// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <h1>
    Welcome
  </h1>
</div>
`;
